grumphp.yml.dist

```diff
@@ -28,9 +36,13 @@ grumphp:
       max_subject_width: 0
 
 parameters:
-  tasks.phpcs.standard: ./vendor/openeuropa/code-review/resources/library-phpcs-ruleset.xml
+  tasks.phpcs.standard: ./vendor/squizlabs/php_codesniffer/src/Standards/PSR12/ruleset.xml
```

Comment:
I think we should have a local ruleset in a resources folder similar to this https://github.com/openeuropa/code-review/blob/2.x/resources/library-phpcs-ruleset.xml because I think it's good practice to harmonize the whitespace